src/Harald.IntegrationTests/Facades/Slack/SlackFacadeTest.cs

```diff
@@ -5,18 +5,20 @@
 using System.Threading.Tasks;
 using Harald.WebApi.Infrastructure.Facades.Slack;
 using Harald.WebApi.Infrastructure.Serialization;
+using Xunit.Priority;
 
 namespace Harald.IntegrationTests.Facades.Slack
 {
+    [TestCaseOrderer(PriorityOrderer.Name, PriorityOrderer.Assembly)]

```

Comment:
If we should make integration tests part of our CI flow, we would need a teardown functionality in my opinion. :)

Reply:
Completely agree.